plugins/out_stackdriver/stackdriver.c

```diff
@@ -839,6 +849,30 @@ static int pack_json_payload(int operation_extracted, int operation_extra_size,
             continue;
         }
 
+        if (validate_key(kv->key, "timestamp", 9)
+            && kv->val.type == MSGPACK_OBJECT_MAP
+            && tms_status == FORMAT_TIMESTAMP_OBJECT) {
```

Comment:
What if `tms_status == INVALID_FORMAT_TIME`? same applies to `timestampSeconds` and `timestampNanos`.

It just seemed that they are inconsistent when key is `time` - but there might be good reason to it - can you let me know?

Reply:
When `tms_status == INVALID_FORMAT_TIME`, we will remove the key `time` in the jsonPayload and use the current time.

There is neither `INVALID_FORMAT_TIMESTAMP_OBJECT` nor `INVALID_FORMAT_TIMESTAMP_DUO`, since if `timestamp` or `timestampSeconds` is invalid, the `tms_status` is set to `TIMESTATMP_NOT_PRESENT`.